apps/src/templates/policy_compliance/NewStudentAccountConsent.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import * as color from '../../util/color';
+
+function permissionGrantedMessage(date) {
+  const grantedDateString = i18n.newStudentAccountConsentValidPermissionGranted(
+    {
+      date: date.toLocaleString(),
+    }
+  );
+  return (
+    <div style={styles.container}>
+      <h2 style={styles.header}>
+        {i18n.newStudentAccountConsentValidHeader()}
+      </h2>
+      <p>
+        <strong>{i18n.newStudentAccountConsentValidPermission()} </strong>
+        <span style={styles.grantDate}>{grantedDateString}</span>
+      </p>
+      <p>{i18n.newStudentAccountConsentValidMessage()}</p>
+      <p>
+        <em>{i18n.newStudentAccountConsentEmailUnknown()}</em>
+      </p>
+    </div>
+  );
+}
+
+function expiredTokenMessage() {
+  return (
+    <div style={styles.container}>
+      <h2 style={styles.header}>
+        {i18n.newStudentAccountConsentExpiredHeader()}
+      </h2>
+      <p>{i18n.newStudentAccountConsentExpiredMessage()}</p>
+      <p>
+        <em>{i18n.newStudentAccountConsentEmailUnknown()}</em>
+      </p>
+    </div>
+  );
+}
+export default function NewStudentAccountConsent(props) {
+  if (props.permissionGranted) {
+    return permissionGrantedMessage(props.permissionGrantedDate);
+  } else {
+    return expiredTokenMessage();
+  }
+}
+
+NewStudentAccountConsent.propTypes = {
+  permissionGranted: PropTypes.bool,
+  permissionGrantedDate: PropTypes.object,
```

Comment:
It was bothering me that there isn't a Date type, but a quick search showed that you can do `PropTypes.instanceOf(Date)`. Editing the storybook seems to validate that this does work.

Reply:
Fixed